src/lib/components/backoffice/MetadataTable/MetadataTable.js

```diff
@@ -13,9 +14,7 @@ export default class MetadataTable extends Component {
         </Table.Cell>
         <Table.Cell>
           {typeof row.value == 'boolean'
-            ? row.value
-              ? 'Yes'
-              : 'No'
+            ? prettyPrintBooleanValue(row.value)
```

Comment:
I am not sure I understand the logic here: isn't it the type already in `row.type`? If yes I would simply:
```
{row.type === 'boolean' ? pretty... : row.value}
```
Remember to use `===` and not `==` ;)

Reply:
No, we don't have the `type` here, we only have it in the extensions fields.